packages/react-components/react-message-bar/library/src/components/MessageBar/useMessageBarStyles.styles.ts

```diff
@@ -15,48 +15,45 @@ const useRootBaseStyles = makeResetStyles({
   gridTemplateColumns: 'auto 1fr auto auto',
   gridTemplateRows: '1fr',
   gridTemplateAreas: '"icon body secondaryActions actions"',
-  paddingLeft: tokens.spacingHorizontalM,
-  border: `${tokens.strokeWidthThin} solid ${tokens.colorNeutralStroke1}`,
-  borderRadius: tokens.borderRadiusMedium,
+  paddingLeft: semanticTokens._ctrlMessageBarPaddingContentAlignDefault,
+  paddingRight: semanticTokens.paddingContentAlignOutdentIconOnSubtle,
+  paddingTop: semanticTokens._ctrlMessageBarPaddingY,
```

Comment:
Let's rename this _ctrlMessageBarPaddingVertical to stick with current naming conventions.